Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:16.04 
+USER root
+
+WORKDIR /tests
+
+ARG STAGE_CONFIG="{'zut':'allures'}"
+
+ENV MOZ_HEADLESS="1"
+ENV RUST_BACKTRACE="full"
+
+RUN apt-get -y update && \
+    apt-get install -y software-properties-common &&\
+    apt-get install -y wget unzip && \
+    apt-get install -y libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev  && \
+    cd /usr/src
+
+RUN apt-get install -y libgtk-3-0 libdbus-glib-1-2 libgtk-3-dev libglib2.0-dev
```

Comment:
Should this be moved up adjacent to L14, or is there significance to doing the previous `cd /usr/src` on L15?